frontend/appflowy_flutter/lib/workspace/application/tabs/tabs_bloc.dart

```diff
@@ -212,6 +221,27 @@ class TabsBloc extends Bloc<TabsEvent, TabsState> {
     }
   }
 
+  Future<void> _expandAncestors(ViewPB view) async {
+    if (isViewExpanded(view.parentViewId)) return;
+    final value = await getIt<KeyValueStorage>().get(KVKeys.expandedViews);
+    final Map expandedViews = value == null ? {} : jsonDecode(value);
```

Comment:
When using `jsonDecode`, it is better to wrap the function in a `try-catch` function to handle potential parsing errors caused by invalid values.